apps/services/university-gateway/infra/university-gateway.ts

```diff
@@ -5,110 +5,109 @@ import {
   UniversityGatewayUniversityOfIceland,
 } from '../../../../infra/src/dsl/xroad'
 
-const namespace = 'university-gateway'
-const imageName = 'services-university-gateway'
+const serviceName = 'services-university-gateway'
+const serviceWorkerName = `${serviceName}-worker`
+const dbName = serviceName.replace(/-/g, '_')
+const namespace = serviceName
+const imageName = serviceName
 
 const postgresInfo = {
-  username: 'university_gateway',
-  name: 'university_gateway',
-  passwordSecret: '/k8s/university-gateway/DB_PASSWORD',
+  username: dbName,
+  name: dbName,
+  passwordSecret: `/k8s/${serviceName}/DB_PASSWORD`,
 }
 
-export const serviceSetup =
-  (): ServiceBuilder<'services-university-gateway'> => {
-    return service('services-university-gateway')
-      .serviceAccount('services-university-gateway')
-      .namespace(namespace)
-      .image(imageName)
-      .command('node')
-      .redis()
-      .args('main.js')
-      .resources({
-        limits: { cpu: '200m', memory: '384Mi' },
-        requests: { cpu: '50m', memory: '256Mi' },
-      })
-      .env({
-        IDENTITY_SERVER_CLIENT_ID: '@island.is/clients/university-gateway',
-        IDENTITY_SERVER_ISSUER_URL: {
-          dev: 'https://identity-server.dev01.devland.is',
-          staging: 'https://identity-server.staging01.devland.is',
-          prod: 'https://innskra.island.is',
+export const serviceSetup = (): ServiceBuilder<typeof serviceName> => {
+  return service(serviceName)
+    .serviceAccount(serviceName)
+    .namespace(namespace)
+    .image(imageName)
+    .command('node')
+    .redis()
+    .args('main.js')
+    .resources({
+      limits: { cpu: '200m', memory: '384Mi' },
+      requests: { cpu: '50m', memory: '256Mi' },
+    })
+    .env({
+      IDENTITY_SERVER_CLIENT_ID: '@island.is/clients/university-gateway',
+      IDENTITY_SERVER_ISSUER_URL: {
+        dev: 'https://identity-server.dev01.devland.is',
+        staging: 'https://identity-server.staging01.devland.is',
+        prod: 'https://innskra.island.is',
+      },
```

Comment:
Is this needed?

Reply:
This seems to be defined in a similar way for most other services. [Endorsement System for example](https://github.com/island-is/island.is/blob/4f6728fa816c37d5720b184e757165aed9898f27/apps/services/endorsements/api/infra/endorsement-system-api.ts#L46)